src/main/kotlin/no/nav/syfo/aktivermelding/ArenaMeldingService.kt

```diff
@@ -0,0 +1,58 @@
+package no.nav.syfo.aktivermelding
+
+import java.time.LocalDate
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+import java.time.format.DateTimeFormatter
+import no.nav.syfo.aktivermelding.arenamodel.Aktivitetskrav8UkerMelding
+import no.nav.syfo.aktivermelding.arenamodel.N2810
+import no.nav.syfo.aktivermelding.arenamodel.N2820
+import no.nav.syfo.aktivermelding.arenamodel.N2830
+import no.nav.syfo.aktivermelding.arenamodel.N2840
+import no.nav.syfo.aktivermelding.arenamodel.tilMqMelding
+import no.nav.syfo.aktivermelding.mq.ArenaMqProducer
+import no.nav.syfo.log
+import no.nav.syfo.model.AKTIVITETSKRAV_8_UKER_TYPE
+import no.nav.syfo.model.PlanlagtMeldingDbModel
+
+class ArenaMeldingService(
+    private val arenaMqProducer: ArenaMqProducer
+) {
+    private val dateFormat = "ddMMyyyy"
+    private val dateTimeFormat = "ddMMyyyy,HHmmss"
+
+    fun sendPlanlagtMeldingTilArena(planlagtMeldingDbModel: PlanlagtMeldingDbModel) {
+        if (planlagtMeldingDbModel.type == AKTIVITETSKRAV_8_UKER_TYPE) {
+            arenaMqProducer.sendTilArena(til8Ukersmelding(planlagtMeldingDbModel, OffsetDateTime.now(ZoneOffset.UTC)).tilMqMelding())
```

Comment:
Blir dette riktig tidspunkt for akkurat nå, eller blir det to timer feil?

Reply:
Det er riktig tidspunkt, men det kommer jo litt ann på hva arena forventer å få inn

Reply:
Arena forventer å få hva klokken er i Oslo akkurat nå :)

Reply:
Ser at tidspunktet blir feil. Setter Oslo-tid :)